snap/hooks/default-configure

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+TAG="$SNAP_NAME.default-configure"
```

Comment:
What's the use of the tag if it isn't used for all the log messages? Some of the logs go to syslog, while some others are printed to standard output.

Reply:
These tags are added here to maintain consistency with the existing tag in the install hook. Additionally, they helped during testing. From syslog, it can be observed that the default-configure hook has been called.

The logs could be modified to all go to syslog. Should all logs go to standard output? If so, the tag in the install hook should also be removed for consistency.

Reply:
These tags are added here to maintain consistency with the existing tag in the install hook. Additionally, they helped during testing. From syslog, it can be observed that the default-configure hook has been called.

The logs could be modified to all go to syslog. Should all logs go to standard output? If so, the tag in the install hook should also be removed for consistency.

Reply:
Please refactor and use logger for all messages. The errors can be sent to syslog as standard error (`logger --stderr ...`). See https://man7.org/linux/man-pages/man1/logger.1.html